addons/website_event_track/models/event_track.py

```diff
@@ -396,7 +396,7 @@ def _compute_track_time_data(self):
         UTC as we compute only time deltas here. """
         now_utc = utc.localize(fields.Datetime.now().replace(microsecond=0))
         for track in self:
-            if not track.date:
+            if not (track.date or track.date_end):
```

Comment:
:red_circle:  If either is present
:green_circle:  If neither is missing
```suggestion
            if not (track.date and track.date_end):
```

Same below